### DISCUSSION TITLE
Getting started with MTEX for EBSD analysis Rev6

### DISCUSSION THREAD

**Discussion by jhiscocks** [mtex-toolbox/mtex#1133] [Announcements]
I'm pleased to announce that the updated (and much improved) version of my MTEX manual is now available online.  With a drastic increase in the number of pictures and code examples, I hope that this will make MTEX more accessible to new users of all types.  Available for download to all at my researchgate page below:

[Getting started with MTEX for EBSD analysis Rev6](https://www.researchgate.net/publication/353330126_Getting_Started_with_MTEX_for_EBSD_analysis_Rev6)

Best Regards, Jessica Hiscocks

**Comment by makeabhishek** [comment]
Thanks, Jessica for your efforts. This will certainly help the MTEX users.

Regards
Abhishek
